src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -126,6 +126,11 @@ public void initializeAuthedSpeechlet() throws URISyntaxException {
         authedSpeechlet.setUserData(testUserData);
     }
 
+    @Before
+    public void resetFactoryObjects() {
+        session = Session.builder().withUser(testUser).withSessionId("test-session-id").build();
```

Comment:
We should probably increment and append a counter to the session ID so the actual session ID is different between each test execution.